tests/unit_tests/gui/test_gui_load.py

```diff
@@ -209,13 +209,13 @@ def test_that_run_dialog_can_be_closed_after_used_to_open_plots(qtbot):
     args_mock = Mock()
     args_mock.config = str(config_file)
 
+    ert_config = ErtConfig.from_file(str(config_file))
+    enkf_main = EnKFMain(ert_config)
     with Storage.init_service(
         res_config=args_mock.config,
```

Comment:
possible inconsistency with line 212 - why not use the `ert_config` from line 212?